src/main/java/com/internet/shop/dao/GenericDao.java

```diff
@@ -0,0 +1,16 @@
+package com.internet.shop.dao;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface GenericDao<T, V> {
```

Comment:
this was done in the previous task. Have you done your pull/push/merge/rebase correctly?